website/views.py

```diff
@@ -98,11 +97,6 @@ def _render_nodes(nodes, auth=None, show_path=False):
 
 @collect_auth
 def index(auth):
-    """Redirect to dashboard if user is logged in, else show homepage.
-
-    """
-    if auth.user:
-        return redirect(web_url_for('home'))
     return {}
```

Comment:
```
(Pdb) l
1592                if rv is not None:
1593                    return rv
1594    
1595            # default method did not match or a specific method is passed,
1596            # check all and go with first result.
1597 ->         for rule in self.map._rules_by_endpoint.get(endpoint, ()):
1598                if rule.suitable_for(values, method):
1599                    rv = rule.build(values, append_unknown)
1600                    if rv is not None:
1601                        return rv
1602    
(Pdb) 
1603        def build(self, endpoint, values=None, method=None, force_external=False,
1604                  append_unknown=True):
1605            """Building URLs works pretty much the other way round.  Instead of
1606            `match` you call `build` and pass it the endpoint and a dict of
1607            arguments for the placeholders.
1608    
1609            The `build` function also accepts an argument called `force_external`
1610            which, if you set it to `True` will force external URLs. Per default
1611            external URLs (include the server name) will only be used if the
1612            target URL is on a different subdomain.
1613    
(Pdb) values
{}
(Pdb) method
u'GET'
(Pdb) r
--Return--
> /Users/whit537/workbench/cos/osf.io/env/lib/python2.7/site-packages/werkzeug/routing.py(1601)_partial_build()->(u'', u'/')
-> return rv
(Pdb) rule
<Rule '/' (HEAD, OPTIONS, GET) -> OsfWebRenderer__index>
(Pdb)
```